Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -864,6 +874,10 @@ void CGameListCtrl::OnRightClick(wxMouseEvent& event)
 				         selected_iso->GetFileName().substr(selected_iso->GetFileName().find_last_of(".")) != ".wbfs")
 					popupMenu.Append(IDM_COMPRESS_ISO, _("Compress ISO..."));
 			}
+			else if (selected_iso->IsElfOrDol())
+			{
+
+			}
 			else
```

Comment:
Instead of having an empty else-if, you could replace the final `else` with `else if (!selected_iso->IsElfOrDol()))`